src/components/inputs/Checkbox.vue

```diff
@@ -1,15 +1,18 @@
 <template>
-	<fieldset class="checkbox-list">
-		<label v-for="option in options" :key="option.label" class="label">
-			<input
-				v-model="vmodelProxy"
-				type="checkbox"
-				:name="option.label + option.value"
-				:value="option.value"
-			/>
-			{{ option.label }}
-		</label>
-	</fieldset>
+	<div>
+		<span v-if="label" class="label">{{ label }}</span>
```

Comment:
move this into the `fieldset` Tag and use the `<legend>Title</legend>` Tag

https://developer.mozilla.org/de/docs/Web/HTML/Element/fieldset#Beispiele